doc/proposals/TEMPLATE.md

```diff
@@ -1,13 +1,22 @@
 ## <Title of Proposal>
```

Comment:
We should discuss as a team, but I wonder if we should start following the [openshift/enhancement template](https://github.com/openshift/enhancements/blob/master/enhancements/template.md) for all SDK proposals, and make that the standard going forward? We will likely be using that openshift/enhancement template for a large portion of our future proposals anyway, so perhaps its best if we use it across the board for consistency.

Reply:
I really agree with. really makes sense we try to adopt/use the openshift template.